AlphaWallet/Core/UnstoppableDomains/DomainResolutionService.swift

```diff
@@ -37,7 +37,7 @@ extension DomainResolutionService: DomainResolutionServiceType {
         }
 
         return Just(value)
-            .receive(on: Config.backgroundQueue)
```

Comment:
Why do we need to move to main queue here?

Reply:
Running it on background and then delivering it in the main thread introduces a predicable ~8 second delay before the background work starts. I don't know why. But seems a good idea to move it completely to the main thread. Most of the time is just spent waiting for the network. I *think* the way to go is just always do things on the main thread, very few select things need to be on background threads. It just complicates things.

I'm going to modify the code a little before force-pushing though. In the PR, it now calls `.receive(on: Runloop.main)` at the start and end of the chain, it should just need 1 call.

Reply:
ok, got it.  There is a call in some of inner functions `address.nameHash` and its could be good to avoid make it on main thread, might block ui